api/lib/middleware/auth.spec.ts

```diff
@@ -14,62 +14,63 @@ import * as idam from '../../services/idam'
 import * as auth from './auth'
 
 describe('auth', () => {
-    describe('validRoles', () => {
-        it('should return true if a give roles match jui pannel member or judge roles', () => {
-            expect(auth.validRoles(['jui-judge'])).to.equal(true)
-            expect(auth.validRoles(['jui-panelmember'])).to.equal(true)
-        })
-
-        xit('should return false if given roles do not match jui pannel member or judge roles', () => {
-            expect(auth.validRoles(['test'])).to.equal(false)
-        }) // FIXME
-
-    })
+    // commenting out as not clear if restricted roles apply to xui
```

Comment:
is this something that will be addressed later on ? maybe TODO

Reply:
Good point , i'll make it a TODO